testfixtures/tests/test_outputcapture.py

```diff
@@ -24,6 +26,12 @@ def test_stdout_and_stderr(self):
             print('now', file=sys.stderr)
         o.compare("hello\nout\nthere\nnow\n")
 
+    def test_unicode(self):
+        with OutputCapture() as o:
+            print(u'こんにちわ', file=sys.stdout)
+            print(u'アウト', file=sys.stderr)
+        o.compare(u"こんにちわ\nアウト\n")
```

Comment:
Please can you change this to \XX format so the encoding at the top is not required?